hewr/R/directory_utils.R

```diff
@@ -61,3 +68,39 @@ parse_model_run_dir <- function(model_run_dir_path) {
     parse_model_batch_dir(batch_dir)
   ))
 }
+
+
+#' Get forecast directories.
```

Comment:
I think we had been calling these `model_batch` directories?

Reply:
Made consistent in 83a1d65